api-specificatie/openapi.yaml

```diff
@@ -1556,7 +1593,21 @@ components:
           minLength: 1
           maxLength: 40
           example: 'AV-2019-08-19-00'
-        mogelijkOnjuist: 
+        adresseerbaarObjectIdentificaties:
+          type: array
+          items:
+            type: string
+        adresIdentificaties:
+          description: Identificatie(s) van het hoofdadres en/of de nevenadressen
+          items:
+            $ref: "#/components/schemas/AdresIdentificatiesArray"
+          type: array
+        nummeraanduidingIdentificaties:
```

Comment:
Een pand heeft een relatie met één of meer adressen, niet met nummeraanduidingen.

Reply:
Dit is een functionele discussie die met Cathy gevoerd moet worden. Als er een functionele behoefte is om op basis van een nummeraanduding de bijbehorende panden op te halen kan je toch via alle verblijfsobjecten waar deze nummeraanduiding een hoofdadres of nevenadres van is alle bijbehorende panden ophalen. 

Dat is precies hetzelfde als je doet met de adressen. Ik zie in het model dat er geen direcet relatei is, maar dat die via het verblijfsobject loopt. Dan kan je die er toch bij zoeken als provider ?